dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -475,7 +478,7 @@ object LagomPlugin extends AutoPlugin {
 
   private lazy val startKafkaServerTask = Def.task {
     val log = new SbtLoggerProxy(state.value.log)
-    val zooKeeperPort = lagomKafkaZookeperPort.value
+    val zooKeeperPort = lagomKafkaZookeeperPort.value.getOrElse(lagomKafkaZookeperPort.value)
```

Comment:
I don't think this will work as expected, since it has set `lagomKafkaZookeeperPort := Some(2181)` above. It should probably default to `None` so that the default behavior is to fall back to any `lagomKafkaZookeperPort` users have set in their existing builds.